connector/mqtt/vernemq/README.md

```diff
@@ -0,0 +1,301 @@
+
+In **TLS** each device and broker should be provisioned with a private key and corresponding public certificate sign from **CA** (certificate authority) and a root certificate (public certificate of **CA**), the **CA** is include in **PKI** (public key infrastructure). A **PKI**  ([EJBCA](./ejbca)) is a system for the creation, storage, and distribution of digital certificates.
+
+An **ACL** (access-control list) based authorization is provided to manage permissions, so a device can only publish and subscribe to its own topic, the topics are: 
+
+- For publish: ***tenant***_:_***device_id***_/attrs_
+- For subscription: ***tenant***_:_***device_id***_/config_
+
+Where ***tenant*** is the information context separation of dojot and ***device_id*** is a unique identification for the device.  
+The junction (***tenant:device_id***) of *tenant* and  *device_id* must be unique.
+
+See more about [ACL Plugin](vernemq/plugins/dojot_acl_plugin) for VerneMQ.
+
+Also a PKI includes the certificate revocation list (**CRL**), which is a list of certificates that have been revoked before reaching the expiration date of the certificate.
+
+The process of obtaining certificates for client (Fig. 2):
+
+- Creation of entity at EJBCA, usually this unique entity consists of ***tenant***_:_***device_id***.
+
+- Obtaining a public certificate for the client, which happens with the creation of the CSR (Certificate Signing Request, it usually contains the public key, identifying information such as a hostname and unique identification) and its submission to obtain a public certificate.
+
+- Obtaining the root certificate (public certificate of **CA**).
+
+![image](http://www.plantuml.com/plantuml/proxy?src=https://raw.githubusercontent.com/dojot/dojot/epic-100kMqttDevices/connector/mqtt/vernemq/docs/plant_uml/mqtt/seq_sec_client)
+
+Fig. 2 - Client retrives certificates from PKI (EJBCA)
+
+The step by step on how to get a certificate for a client will be explained later, see [here](##how-to-get-a-certificate-for-a-client-to-use-with-dojot-vernemq).
+
+The process of obtaining certificates for VerneMQ, K2V Brige and K2V-bridge (services) instances follows the same steps as for the client, with some more followed by (Fig. 3):
+
+- At each defined time (CHECK_EXPIRATION_TIME), it's checked if the root certificate and public certificate of the service instance will expire in the next CHECKEND_EXPIRATION_SEC seconds.
+
+- At each defined time (CHECK_BROKER_CERT_REVOKED_TIME), it's checked if the public certificate of the service instance has been revoked.
+
+- CRL updated the CRL certificate every time by setting in CRL_UPDATE_TIME
+
+
+![image](http://www.plantuml.com/plantuml/proxy?src=https://raw.githubusercontent.com/dojot/dojot/epic-100kMqttDevices/connector/mqtt/vernemq/docs/plant_uml/mqtt/seq_sec_service1)
```

Comment:
You can look the image before merge here
http://www.plantuml.com/plantuml/proxy?src=https://raw.githubusercontent.com/mprevide/dojot/readme_mqtt/connector/mqtt/vernemq/docs/plant_uml/mqtt/seq_sec_service1